src/plugins/intel_cpu/src/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -924,23 +931,31 @@ void MKLDNNDeformableConvolutionNode::createPrimitive() {
     jcp.nb_oc_blocking = !mayiuse(cpu::x64::avx2) ? 2 : 4;
 
     jcp.nthr = dnnl_get_max_threads();
+}
 
-    if (enforceRef) {
-        return;
-    } else if (mayiuse(cpu::x64::avx512_common)) {
+MKLDNNDeformableConvolutionNode::DefConvJitExecutor::DefConvJitExecutor(const DefConvAttr &defConvAttr,
+                            const std::vector<std::shared_ptr<BlockedMemoryDesc>> &descVector,
+                            const std::vector<ptrdiff_t> &padL) :
```

Comment:
Why `padL` is not part of `DefConvAttr`?

Reply:
Because of autoPad case. In the previous version defConvAttr was a holder of parameters instead of jcp itself, so it wasn't included in to-be-cached executor as mutable parameter. Now it is really better to include padL in defConvAttr.